src/Npgsql/NpgsqlConnectionStringBuilder.cs

```diff
@@ -777,13 +777,17 @@ public int ConnectionPruningInterval
     /// <summary>
     /// The total maximum lifetime of connections (in seconds). Connections which have exceeded this value will be
     /// destroyed instead of returned from the pool. This is useful in clustered configurations to force load
-    /// balancing between a running server and a server just brought online.
+    /// balancing between a running server and a server just brought online. It can also be useful to prevent
+    /// runaway memory growth of connections at the PostgreSQL server side, because in some cases very long lived
+    /// connections slowly consume more and more memory over time.
+    /// Defaults to 3600 seconds (1 hour).
```

Comment:
Need to also fix the `<value>` doc comment just below.

Reply:
done